src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -198,10 +198,7 @@ describe('ReactDOMComponent', function() {
 
       expect(console.error.argsForCall.length).toBe(1);
       expect(console.error.argsForCall[0][0]).toEqual(
-        'Warning: `span` was passed a style object that has previously been ' +
-        'mutated. Mutating `style` is deprecated. Consider cloning it ' +
-        'beforehand. Check the `render` using <span>. Previous style: ' +
-        '{fontSize: NaN}. Mutated style: {fontSize: NaN}.'
+        'Warning: You passed NaN as a value for fontSize. Are you sure this is intentional?'
```

Comment:
It might not be immediately obvious to a user that we're talking about css styles here, it might be good to mention that it's a css style.  Also, I'm not sure any of our other warnings end in a question mark.  Maybe something like:

 > \`NaN\` is an invalid value for the \`fontSize\` css style property